products/mariadb.md

```diff
@@ -56,9 +56,15 @@ auto:
 
 # Extend/update regex in section above
 releases:
+-   releaseCycle: "11.7"
+    releaseDate: 2025-02-13
```

Comment:
```suggestion
    releaseDate: 2024-09-25
```

11.7 series started with : https://mariadb.com/kb/en/mariadb-11-7-0-release-notes/
releasedate tag should show the date of any cycle's first release

Reply:
fair enough. 👍

Reply:
@usta I am gonna revert this change (can you create a separate PR to update all the releaseDates for the cycles, and I think we are tracking stable release rather than alpha release from beginning) cc @captn3m0 @marcwrobel